Assignment_05.rb

```diff
@@ -55,8 +55,19 @@ def total_time_in_minute
   end
 
   # This method takes total time used for a task and returns the in percentage form for a day
-  def occupation_percecntage_day_wise(total_time_in_minute)
-    (total_time_in_minute.to_f / (24 * 60) * 100).round(2)
+  def occupation_percecntage(total_time_in_minute, period)
```

Comment:
can you please rename the method as `occupancy_percentage`

Reply:
typo corrected